### PR TITLE
Oled example

### PR DESCRIPTION
**Pull request by gautam-dev-maker**
* Add Oled Example in the SRA-BOARD-COMPONENT
* Update github actions for oled-example

### CODE REVIEW

**Review by VedantParanjape — APPROVED** [review]

File: .github/workflows/test-build-app.yaml [hunk 1]
```diff
@@ -9,9 +9,17 @@ jobs:
       image: espressif/idf:release-v4.1
     strategy:
       matrix:
-        test-apps: [bar_graph, lsa, motor_driver_normal, motor_driver_parallel, mpu6050, servos, switches]
-    steps:  
-      - uses: actions/checkout@v2
+        test-apps: [bar_graph, lsa, motor_driver_normal, motor_driver_parallel, mpu6050, servos, switches, oled]
+    steps:
+      - name: Setup Github Actions
+        uses: actions/checkout@v1
```
Comment (VedantParanjape) [inline .github/workflows/test-build-app.yaml:15]: why did you downgrade to v1 ?

Comment (VedantParanjape) [reply .github/workflows/test-build-app.yaml:15]: https://remarkablemark.org/blog/2021/02/21/github-actions-checkout-submodules/ Please follow the approach 1 here with checkout@v2

Comment (SAtacker) [reply .github/workflows/test-build-app.yaml:15]: Please check [log](https://github.com/gautam-dev-maker/sra-board-component/actions/runs/1859017549) of the [workflow](https://github.com/gautam-dev-maker/sra-board-component/actions/runs/1859017549/workflow) with the suggestion 1 given.
Suggestion 2 [workflow](https://github.com/gautam-dev-maker/sra-board-component/actions/runs/1859109916/workflow) , [log](https://github.com/gautam-dev-maker/sra-board-component/actions/runs/1859109916)

Comment (VedantParanjape) [reply .github/workflows/test-build-app.yaml:15]: @SAtacker I had encountered this issue, just purge and install new git 

```bash
sudo add-apt-repository ppa:git-core/ppa -y
sudo apt-get update
sudo apt-get install git -y
```

Comment (VedantParanjape) [reply .github/workflows/test-build-app.yaml:15]: Also, I had encountered this issue on the website as well. This works fine here: https://github.com/SRA-VJTI/sra-vjti.github.io/blob/main/.github/workflows/add-intern-exp.yaml

Comment (VedantParanjape) [reply .github/workflows/test-build-app.yaml:15]: https://github.com/actions/virtual-environments/blob/c253e7f4d192e492988596167a1d461e99698c13/images/linux/Ubuntu2004-README.md#tools

According to this ubuntu-latest should have git 2.30.2. I guess the issue is with permission on gautam's fork. Please remove the changes and let's see if it fails in this repo.

**Comment by VedantParanjape** [comment]
@gautam-dev-maker updates ?

**Comment by gautam-dev-maker** [comment]
I have reverted the commits regarding the github actions.

**Comment by VedantParanjape** [comment]
@gautam-dev-maker you still need to change the yaml file to include oled folder of the examples

**Comment by VedantParanjape** [comment]
@gautam-dev-maker you also need to enable submodule in git checkout https://github.com/SRA-VJTI/sra-board-component/pull/55#discussion_r813311685

CC: @SAtacker

**Review by VedantParanjape — APPROVED** [review]

**Pull request merged**